libs/features/personalization/personalization.js

```diff
@@ -448,6 +449,10 @@ function modifySelectorTerm(termParam) {
     term = updateEndNumber(endNumber, term);
     return term;
   }
+  if (containerSelectors.includes(startText)) {
```

Comment:
If we just check for `'marquee-container'` maybe we shouldn't use an array 🤔
```suggestion
  if (startText === 'marquee-container') {
```
P.S. You'll also save 1 line like this